src/actions/fetchProfileAction.js

```diff
@@ -0,0 +1,27 @@
+import { REQUEST_PROFILES, REQUEST_PROFILES_SUCCESS, REQUEST_PROFILES_FAILURE } from './types';
+import RetrieveUserProfilesAPI from '../api/fetchProfileApi';
+
+export const requestUserProfilesSuccess = (payload = {}) => ({
+  type: REQUEST_PROFILES_SUCCESS,
+  payload,
+});
+
+export const requestUserProfilesFailure = (payload = {}) => ({
```

Comment:
This can be improved in the future. No need to initialize in action. This can be done in reducer.